apps/container/src/components/ConceptsPieChart.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="chartWrapper">
+    <Chart :options="chartOptions"></Chart>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { conceptsColor } from 'consts';
+import { Chart } from 'highcharts-vue';
+import { computed, PropType } from 'vue';
+
+type Grades = {
+  [x: string]: number;
+};
```

Comment:
Mais simples o Record
```suggestion
type Grades = Record<string, number>
```